src/helpers/__mocks__/dimensions.ts

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
there is an existing __mocks__ folder in the root directory, would you mind trying moving this file to there so we centralize our mocks there? Thanks!